constants/index.ts

```diff
@@ -76,3 +76,61 @@ export const DISCORD_URL =
   process.env.NEXT_PUBLIC_DISCORD_URL || 'https://discord.com/invite/QFkNsjTkzD';
 export const GITHUB_URL =
   process.env.NEXT_PUBLIC_GITHUB_URL || 'https://github.com/SwaprHQ/presagio';
+
+const formatEventName = (eventName: string) =>
+  eventName.split(' ').join('-').toLocaleLowerCase();
+export const FA_EVENTS = {
+  FOOTER: {
+    CONNECT: {
+      DISCORD: 'click/footer-connect_discord',
```

Comment:
Why do we have dashes and underscores?

Reply:
I used them to express a hierarchy, in my head:

`$eventTrigger/$path-to-event_$event-detail`